### PR TITLE
fix: [CDS-55698]:configure option fixed for jenkins jobname

### PR DESCRIPTION
**Pull request by satyam-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by satyam-harness** [comment]
retrigger titlecheck

**Comment by satyam-harness** [comment]
retrigger jest

File: src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/JenkinsArtifactSource/JenkinsArtifactSource.tsx [hunk 1]
```diff
@@ -600,7 +605,7 @@ const Content = (props: JenkinsRenderContent): React.ReactElement => {
                       addClearBtn: false,
                       itemRenderer: jobNameRenderer
                     },
-                    allowableTypes
+                    allowableTypes: [MultiTypeInputType.FIXED]
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/JenkinsArtifactSource/JenkinsArtifactSource.tsx:608]: Are we supporting only `FIXED` value for this field in run pipeline form?

Comment (satyam-harness) [reply src/modules/75-cd/components/PipelineSteps/K8sServiceSpec/ArtifactSource/JenkinsArtifactSource/JenkinsArtifactSource.tsx:608]: yes

**Review by pankajsingh-harness — APPROVED** [review]

**Pull request merged**